### ISSUE TITLE
Heimdall loses it's settings every reboot

### ISSUE THREAD

**Issue by synegic** [daneedk/com.uc.heimdall#4]
Hello there,

First of all, thanks for building this app. It's a great idea and certainly works better than creating a surveillance mode by flows. It also seems to function great, especially for a version 0.1.x.

I am, however, experiencing a problem. It seems like Heimdall loses all app settings after a reboot. I have to select modes and input the delay durations again (all are empty), nothing gets logged, etc. This in combination with the Zigbee bug playing up since 1.5.6 requiring a daily PTP makes it unusable at this time..

My other apps seem fine and are remembering their settings. I'm on 1.5.6-rc.8. Installed Heimdall as soon as it came out. Flow cards and all remain untouched and working after reboot.

What's going on? :-)

**Comment by daneedk** [comment]
Hello,

I can't really tell what's going on unfortunately and I can also not reproduce it. You are the first that runs into this problem (or at least reports it)
Can you please file a report here: https://github.com/athombv/homey/issues and post the issue number here so I can add relevant information.

**Comment by synegic** [comment]
Hi daneedk,

Sorry for the long delay. After I cleaned out my Insights storage (which indicated 99% full), it seems to remember it's settings again. Although Homey said I had 100MB left on storage, I could imagine this being a problem. So I don't think it was related to Heimdall, but rather the Homey OS itself (with Heimdall being the special case because it's the last app I installed).

False alarm I assume!
